.github/workflows/build-push.yml

```diff
@@ -9,59 +9,53 @@ on:
 
 jobs:
   build-publish-proto-dependencies-focal-fossa:
-    runs-on: ubuntu-latest
     name: Build and publish proto dependencies image on ubuntu 20.04
-    steps:
-      - name: Checkout Repository
-        uses: actions/checkout@v2
-
-      - name: Build and Push
-        uses: ./.github/actions/build-push
-        with:
-          image: "Protocol dependencies"
-          base_tag: 20.04
-          secret: ${{ secrets.GITHUB_TOKEN }}
+    uses: aica-technology/.github/.github/workflows/build-push-multi-arch.yml@ed4ba3f4327b2a61948cde7b4e9410363114c478
```

Comment:
Just to check, is only the commit ref `ed4ba3f4` allowed, or can it be a branch / tag? The latter would be preferable for clarity.

Reply:
It can be a branch or a tag yes. I prefer tag or ref though, because otherwise we might break stuff by changing the .github workflow without updating all workflows that use it..

Reply:
A tag would be greatly preferred for me. Start with v0.1 for the current version and we can go incrementally from there.